packages/utils/src/error.ts

```diff
@@ -0,0 +1,14 @@
+let set: Set<string>;
+if (process.env.NODE_ENV !== 'production') {
+  set = new Set<string>();
+}
+
+export function error(...messages: string[]) {
```

Comment:
Is the variadic parameter really necessary? Looks like the use-cases could be just string templates.

Reply:
It matches the `console` APIs